src/plugins/console/public/lib/autocomplete/components/param_component.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Term } from '../types';
+import { ConstantComponent } from './constant_component';
+
+export class ParamComponent extends ConstantComponent {
+  description: string | unknown;
```

Comment:
Yes, after researching the ```description``` variable I've found out that it is the same as in https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2322c53d10664d68c7d2ee21a86a4e29d1919d9c/src/plugins/console/server/services/spec_definitions_service.ts#L43-L53
Here we can see that for description ```any``` is used, also here is the examples of description variable 
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2322c53d10664d68c7d2ee21a86a4e29d1919d9c/src/plugins/console/public/lib/autocomplete/__jest__/url_params.test.js#L95-L98
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2322c53d10664d68c7d2ee21a86a4e29d1919d9c/src/plugins/console/public/application/models/sense_editor/__tests__/integration.test.js#L1013-L1016
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2322c53d10664d68c7d2ee21a86a4e29d1919d9c/src/plugins/console/server/lib/spec_definitions/json/generated/cat.allocation.json#L3-L23
I have seen other examples of big complex ```description```s, but I'm not sure that they are the same as this one.

Overall I concluded that it can be a big complex object and we can't be 100% sure how it will look like.